tutorials/optical_tactile_plugin.md

```diff
@@ -1,28 +1,23 @@
 \page opticaltactileplugin Optical Tactile Plugin
 
-This plugin implements a tactile sensor from a depth camera and a contact sensor. 
+This plugin simulates an optical tactile sensor from a depth camera and a contact sensor. 
 Given the increasing number of this kind of sensors and their use cases, this 
-plugin is the first step towards precisely evaluating this technology and its 
-possible applications before being deployed in the real world.
+plugin is the first step towards data collection and sanity check of algorithms
+involving this type of sensors.
 
 Currently, the plugin is required to work within models that have one link, one
-contact sensor and one depth camera. In order to simulate the behaviour of a
-deformable membrane that returns the contact forces of the object being touched, 
-we merge the information coming from this two different sensors. Vectors normal to 
-the surfaces being touched are computed from the values coming from the depth camera.
-Next, this values are merged with the information returned by the contact sensor, i.e.
-force magnitudes, penetration and depth.
+contact sensor and one depth camera. In order to simulate the behaviour of the
+contact surface, which returns the contact forces of the object being touched,
+we merge the information coming from these two different sensors. Contact normals to 
+the surfaces being touched are computed from the depth image. (TODO) Next, these
+values are merged with the information returned by the contact sensor, i.e. force 
+magnitudes, penetration and depth.
 
-Functionality can be added on top of the information returned by this sensor,
-like object recognition, tactile imaging, precise manipulation or human-robot
-interaction. However, this is something not currently supported, and the plugin
-only returns the information computed.
+The plugin allows the user to visualize the contact normals of the objects in 
+the proximity of being touched. In the next image, we can see these normals 
+when we touch the ring and the top surface of a can of coke:
 
-The plugin allows the user to visualize the surface vectors of the objects being touched.
-In the next image, we can see these vectors when we touch the ring and the top
-surface of a can of coke:
-
-<img src="https://github.com/ignitionrobotics/ign-gazebo/blob/master/tutorials/files/optical_tactile_plugin/coke_can.png"/>
+<img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/master/tutorials/files/optical_tactile_plugin/coke_can.png"/>
```

Comment:
Is it possible to show the curved side of the can with normals in several directions? The interesting part of the can is really the curved surface and not the flat top and bottom.

Reply:
1dec57f